src/content/docs/dropins/payment-services/containers/credit-card.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: CreditCard container
+description: Configure the CreditCard container to manage and display the available credit card fields during a checkout.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The `CreditCard` container allows a shopper to place an order with credit card details during the checkout process. You can configure it to manage and display the available credit card fields for a payment method.
+
+## CreditCard configurations
+
+The `CreditCard` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['apiUrl', 'string', 'Yes', 'The URL to the Adobe Commerce GraphQL endpoint, such as "https://example.com/graphql".'],
+    ['getCartId', 'function', 'Yes', 'Should return a promise that resolves to the shopper`s cart ID.'],
+    ['getCustomerToken', 'function', 'No', 'The `CreditCard` container may send GraphQL requests on behalf of the shopper. This requires GraphQL authorization, which can be performed using authorization tokens or session cookies. For token-based authorization, the "getCustomerToken" function that should return a customer token as a string, or null for guest checkouts.'],
```

Comment:
Lastly, `Req?: No` might not be entirely correct. Strictly speaking, this parameter is conditionally required. 

* Merchant uses session-based GraphQL auth → `getCustomerToken` must not be provided (or must be `undefined`).

* Merchant uses token-based GraphQL auth → `getCustomerToken` must be provided and must return the customer token as a string or `null` if there is no customer logged in (i.e., guest).